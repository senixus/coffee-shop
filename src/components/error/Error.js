import React from "react";
import { Link } from "react-router-dom";

import "./error.scss";

const Error = () => {
  return (
    <div className="error">
      <h1>404</h1>
      <p>page not found</p>
      <Link to="/">Home</Link>
    </div>
  );
};

export default Error;
