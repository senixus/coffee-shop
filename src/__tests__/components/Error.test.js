import { shallow } from "enzyme";
import React from "react";
import Error from "../../components/error/Error";

it("expect to render Error component ", () => {
  const wrapper = shallow(<Error />);

  expect(wrapper.length).toBe(1);
  expect(wrapper).toMatchSnapshot();
});
