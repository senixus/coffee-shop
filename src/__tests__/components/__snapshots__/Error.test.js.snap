// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Error component  1`] = `
<div
  className="error"
>
  <h1>
    404
  </h1>
  <p>
    page not found
  </p>
  <Link
    to="/"
  >
    Home
  </Link>
</div>
`;
